lerna.json

```diff
@@ -37,6 +37,6 @@
   ],
   "npmClient": "yarn",
   "npmClientArgs": [
-    "--no-lockfile"
+    "--pure-lockfile"
```

Comment:
According to https://github.com/yarnpkg/yarn/issues/5847#issuecomment-537521943, `--pure-lockfile` will not modify `yarn.lock` even if we manually update `package.json`. I think can just remove `npmClientArgs` and resort to the default behaviour.

@hzoo BTW `--no-lockfile` was introduced at https://github.com/babel/babel/pull/6440, do you have any concerns about why `--no-lockfile` was applied?

Reply:
haha was so long ago, I assume it was because it was adding a lockfile to every package when you run `make bootstrap` instead of one at the top level?

Reply:
I chose `--pure-lockfile` so that lerna wouldn't create a lock file in every package, but would still read from it. If we want to use lock files everywhere, I'd be fine with removing.

Reply:
Oh I meant that we didn't want lockfiles everywhere so I believe that's why `--no-lockfile` was originally added. yeah maybe pure is the way to go then but yeah need to update manually otherwise.